content/en/docs/components/pipelines/v2/author-a-pipeline/pipelines.md

```diff
@@ -115,11 +115,31 @@ def my_pipeline():
     with dsl.ExitHandler(exit_task=print_status_task):
         fail_op()
 ```
+#### Ignore upstream failure
+Another similiar method to achieve the same process as the exithandler while at the same time allowing tasks to collect output from upstream tasks is the [`ignore_upstream_failure()`][ignore-upstream-failure] configuration from [`pipeline_task`][tasks-configurations]
```

Comment:
Suggested rewording:

The [`ignore_upstream_failure()`][ignore-upstream-failure] configuration from [`pipeline_task`] converts the caller task into an exit task if the pipeline task is unsuccessful. If the caller task has upstream dependencies, this method turns it into an exit handler task and allows tasks to collect outputs from the upstream tasks. If the task has no upstream tasks, either via data exchange or an explicit dependency via `.after()`, this method has no effect.